Source/WebCore/css/StyleProperties.cpp

```diff
@@ -690,6 +692,45 @@ String StyleProperties::fontVariantValue() const
     return result.toString();
 }
 
+String StyleProperties::fontSynthesisValue() const
+{
+    StringBuilder result;
+
+    auto getLonghandValue = [&](CSSPropertyID propertyID) {
+        auto foundPropertyIndex = findPropertyIndex(propertyID);
+        if (foundPropertyIndex == -1)
+            return String();
+
+        auto property = propertyAt(foundPropertyIndex);
+        return property.value()->cssText();
```

Comment:
It's more efficient to do comparisons on the values directly rather than the cssText.